backend/data/src/main/kotlin/io/tolgee/service/apps/AppManifestHttpClient.kt

```
@@ -0,0 +1,74 @@
+    val body =
+      try {
+        restTemplate.execute(url, HttpMethod.GET, null, { response ->
+          if (!response.statusCode.is2xxSuccessful) {
+            throw BadRequestException(
+              Message.APP_MANIFEST_FETCH_FAILED,
+              listOf("unexpected status ${response.statusCode.value()}"),
+            )
+          }
+          readBounded(response.body)
+        })
+      } catch (e: RestClientException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)